index.js

```diff
@@ -255,19 +256,21 @@ if (heroku.dockerBuildArgs) {
       }
     }
 
-    core.setOutput(
-      "status",
-      "Successfully deployed heroku app from branch " + heroku.branch
-    );
+    fs.appendFileSync(process.env.GITHUB_OUTPUT,
```

Comment:
None of these changes are needed.  Just need to update actions/core to version >= 1.10.0.  The implementation of `setOutput` has been changed to be complaint.

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows